src/test/util/event.rs

```diff
@@ -120,7 +120,9 @@ pub struct EventHandler {
     command_events: EventQueue<CommandEvent>,
     pub pool_cleared_events: EventQueue<PoolClearedEvent>,
```

Comment:
I think we can just drop this field now that we have full support for CMAP events.

Reply:
Done.